.gitignore

```diff
@@ -1,6 +1,8 @@
 # Directories #
 /build/
 target/
+.metals/
+.vscode/
```

Comment:
Let's add this after `metastore_db/` to group `.*/` together

Reply:
done

Reply:
reverted as we need .gitignore